CI-CD/CloudBuild/Unified Agent/GoogleCloudBuild.yaml

```diff
@@ -14,5 +14,5 @@ steps:
   - 'WS_WSS_URL=https://saas.mend.io/agent'
   - 'WS_PRODUCTNAME=$PROJECT_ID'
   - 'WS_PROJECTNAME=$BUILD_ID'
-  - 'WS_FILESYSTEMSCAN: false'
+  - 'WS_FILESYSTEMSCAN=false'
```

Comment:
@lukebrogan-mend why is file system scan set to false?

Reply:
on previous example, will remove